### PR TITLE
Unit test for flex sensor.cpp #3091

### PR DESCRIPTION
**Pull request by alxrMironov**
Unit test for flex sensor.cpp by #3091 ticket

### CODE REVIEW

**Comment by alxrMironov** [comment]
Test

File: unit_tests/tests/sensor/test_flex_sensor.cpp [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+#include <gtest/gtest.h>
+
+class FlexSensorForTests : public ::testing::Test
+{
+
+protected:
+
+    FlexSensorForTests () :
+            dut ()
```
Comment (mck1117) [inline unit_tests/tests/sensor/test_flex_sensor.cpp:13]: explicit no-arg constructor is not required

Comment (mck1117) [reply unit_tests/tests/sensor/test_flex_sensor.cpp:13]: I guess no constructor is required at all, the default is plenty here.

File: unit_tests/tests/sensor/test_flex_sensor.cpp [hunk 2]
```diff
@@ -0,0 +1,148 @@
+#include "flex_sensor.h"
+#include "engine_test_helper.h" // Used for timesourcing
```
Comment (mck1117) [inline unit_tests/tests/sensor/test_flex_sensor.cpp:2]: you can replace all these includes with `#include "pch.h"`, except for flex_sensor.h

File: unit_tests/tests/sensor/test_flex_sensor.cpp [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+protected:
+
+    FlexSensorForTests () :
+            dut ()
+    {
+    }
+    ;
+
+    void
+    SetUp () override
```
Comment (mck1117) [inline unit_tests/tests/sensor/test_flex_sensor.cpp:19]: should be `void SetUp() override {`, similar with others

File: unit_tests/tests/sensor/test_flex_sensor.cpp (hunk 2 above)
Comment (alxrMironov) [reply unit_tests/tests/sensor/test_flex_sensor.cpp]: Noted

File: unit_tests/tests/sensor/test_flex_sensor.cpp (hunk 1 above)
Comment (alxrMironov) [reply unit_tests/tests/sensor/test_flex_sensor.cpp]: Noted

**Pull request merged**